examples/table.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable no-console,func-names,react/no-multi-comp */
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Table } from 'antd';
+import ReactDragListView from 'react-drag-listview/src/index.js';
+import './index.less';
+
+class Demo extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      dataSource: [
+        {
+          key: '1',
+          name: '1111',
+          age: 32,
+          address: '10 Downing Street'
+        },
+        {
+          key: '2',
+          name: '2222',
+          age: 42,
+          address: '10 Downing Street'
+        },
+        {
+          key: '3',
+          name: '3333',
+          age: 42,
+          address: '10 Downing Street'
+        },
+        {
+          key: '4',
+          name: '4444',
+          age: 42,
+          address: '10 Downing Street'
+        },
+      ],
+    };
+  }
+
+  render() {
+    const columns = [
+      {
+        title: 'Name',
+        dataIndex: 'name',
+        key: 'name',
+      },
+      {
+        title: 'Age',
+        dataIndex: 'age',
+        key: 'age',
+      },
+      {
+        title: 'Address',
+        dataIndex: 'address',
+        key: 'address',
+      },
+      {
+        key: '__drag__',
+        render: () => (
+          <a>drag</a>
+        ),
+      },
+    ];
+    const dragProps = {
+      onDragEnd: (fromIndex, toIndex) => {
+        const { dataSource } = this.state;
+        const item = dataSource.splice(fromIndex, 1)[0];
+        dataSource.splice(toIndex, 0, item);
+        this.setState({ dataSource });
+      },
+      handleSelector: 'a',
```

Comment:
The dragged line will select the expanded row， add prop   ```nodeSelector: ".ant-table-row",```

Reply:
add prop `nodeSelector: '.ant-table-row'` still throw errors

Reply:
I have tested it and found no problems. Can you provide a test link for trouble?

Reply:
https://codepen.io/anon/pen/BvvbWv

move the last line, and you can see the error

Reply:
I recommend adding attributes based on the PR you submitted.     It has not been released yet！